plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/r4/provider/CareGapsProvider.java

```diff
@@ -208,17 +210,19 @@ public Parameters careGapsReport(RequestDetails theRequestDetails,
 			throw new NotImplementedException("Non subject parameters have not been implemented.");
 		}
 
+		StopWatch watch = StopWatch.createStarted();
```

Comment:
Need to remove this since it's not being used.

Reply:
Done!